python/lsst/ts/m2/csc.py

```diff
@@ -525,17 +545,25 @@ async def do_enable(self, data: salobj.BaseMsgType) -> None:
                     retry_times=self.ilc_retry_times,  # type: ignore[arg-type]
                 )
 
-            except RuntimeError:
-                # TODO: Publish the list of failed ILCs after the DM-40146 is
-                # done.
+            except RuntimeError as error:
                 await self._basic_cleanup_and_power_off_motor()
+
+                error.add_note("Power-off the motor.")
```

Comment:
Not sure I understand what is goin on here. Can you clarify a bit?

If I understood it correctly you check if the ILC modes are enabled and if it is not you try to enable them. If that fails you power off the motors. If this is correct, maybe expand the error message to something like:

```
Failed to enable ILCs. Powering off motors.
```

would be good if you could add some suggestion of what could be done to work around or troubleshoot the problem.

Reply:
Updated in the commit: [cbe9944](https://github.com/lsst-ts/ts_m2/pull/56/commits/cbe99449d8ee63e2b8c583e451cee4ffd355c7ff). Thanks!